Objects/dictobject.c

```diff
@@ -4122,6 +4122,23 @@ _PyDictView_New(PyObject *dict, PyTypeObject *type)
     return (PyObject *)dv;
 }
 
+static PyObject *
+dictview_mapping(PyObject *view)
```

Comment:
It should have signature compatible with `getter`:
```
static PyObject *
dictview_mapping(PyObject *view, void *Py_UNUSED(ignored))
```

Reply:
Fixed in https://github.com/python/cpython/pull/20876